src/fmu/tools/properties/swfunction.py

```diff
@@ -331,4 +309,11 @@ def compute(self, compute_method: str = "integrated") -> xtgeo.GridProperty:
             "Bug: Grid mask and Sw mask are not equal, contact developer!"
         )
 
-        return self._sw
+        output_props = [self._sw, self.htop, self.hbot, self.hcenter]
+        output_props_name = ["SW", "HTOP", "HBOT", "HCENTER"]
+
+        for prop, prop_name in zip(output_props, output_props_name):
+            assert isinstance(prop, xtgeo.GridProperty)
+            prop.name = self.tag + prop_name
+
+        return tuple(output_props)
```

Comment:
Instead of returning a tuple, could a dict return be more user friendly and also more flexible in the future?
```
sw = sw_obj.compute("integrated")["SW"]
```